tests/ParserTest.php

```diff
@@ -1161,13 +1161,16 @@ public function commentExtracting(): void
      */
     public function flatCommentExtracting(): void
```

Comment:
We're planning to split the tests and testcases into smaller, more focused tests. To make this easier for us, please don't expand the test. Instead, could you please duplicate it into two separate tests, one with a single comment and one with multiple? Thanks!